lib/home/home_page_content.dart

```diff
@@ -12,10 +12,28 @@ class HomePageContent extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final widgets = sections.map((section) {
-      if (section is CarouselData) {
+      if (section is SpacerData) {
```

Comment:
Hmm I guess eventually we would move this to a more generic CMS display function?

Reply:
I haven't thought this far but yes, there is room for improvement